lib/Yield/Support/GraphLayout/SugiyamaStyle/NodeRanking.h

```diff
@@ -16,7 +16,7 @@ RankContainer rankNodes(InternalGraph &Graph, int64_t DiamondBound);
 
 /// Updates node ranking after the graph was modified.
 ///
-/// Garanties ranking consistency i.e. that each node has
+/// Guaranties ranking consistency i.e. that each node has
```

Comment:
Guarantees

Reply:
I think this was lost across force-pushes. Please double check

Reply:
Re-cherry-picked that commit.